modules/katamari/src/test/ts/module/ephox/katamari/test/arb/ArbDataTypes.ts

```diff
@@ -14,7 +14,7 @@ export const arbResultValue = <A, E> (arbA: Arbitrary<A>): Arbitrary<Result<A, E
 export const arbResult = <A, E> (arbA: Arbitrary<A>, arbE: Arbitrary<E>): Arbitrary<Result<A, E>> =>
   fc.oneof(arbResultError<A, E>(arbE), arbResultValue<A, E>(arbA));
 
-export const arbOptionalNone = <T> () => fc.constant(Optional.none());
+export const arbOptionalNone = <T> () => fc.constant(Optional.none<T>());
```

Comment:
What's the reasoning behind removing this type annotation?

Reply:
Not sure what you mean sorry? I've added it because it was previously missing and the `<T>` generic was never used.

Reply:
I must be tired. Ignore me.